docs/how_to_guides/how_to_setup_simple_chemistry.rst

```diff
@@ -0,0 +1,448 @@
+--------------------------------------------------
+
+You will always be required to provide a configuration dictionary for thermodynamic
+properties of the chemical species of interest in your process. At a minimum, the
+**thermo-properties** dictionary MUST contain the following keys...
+
++----------------------+-------------------------------------------------------------------------------------------+
+|     Key              |  Description                                                                              |
++======================+===========================================================================================+
+| components           | dictionary containing the chemical species of interest and their properties               |
++----------------------+-------------------------------------------------------------------------------------------+
+| phases               | dictionary containing the phases (gas, liquid, etc) for the system and its species        |
++----------------------+-------------------------------------------------------------------------------------------+
+| state_definition     | IDAES object which defines how inlet/outlet ports define the initial state of the system  |
++----------------------+-------------------------------------------------------------------------------------------+
+| base_units           | dictionary containing the base units the model will use for all equations in the system   |
++----------------------+-------------------------------------------------------------------------------------------+
+
+.. note::
+
+    Depending on your system, there may be additional keys in the **thermo-properties**
+    configuration dictionary that may be required. For instance, multi-phase problems
+    also require you to define how the model should represent the phase equilibria
+    between the system's phases. An in depth discussion of all options and methods
+    is beyond the scope of this guide. For additional information, refer to the IDAES
+    documentation (`GenericProperties`_).
+
+
+Example thermo-properties configuration dictionary
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this example, we will define a simple thermo-properties configuration dictionary
+for a chemical system that contains only water.
+
+.. code-block:: python
+
+    # Importing the object for units from pyomo
+    from pyomo.environ import units as pyunits
+
+    # Imports from idaes core
+    from idaes.core import AqueousPhase
+    from idaes.core.components import Solvent, Cation, Anion
+
+    # Imports from idaes generic models
+    import idaes.generic_models.properties.core.pure.Perrys as Perrys
+    from idaes.generic_models.properties.core.pure.ConstantProperties import Constant
+    from idaes.generic_models.properties.core.state_definitions import FTPx
+    from idaes.generic_models.properties.core.eos.ideal import Ideal
+
+    # Configuration dictionary
+    thermo_config = {

```

Comment:
I think there may be an error somewhere in `thermo_config` because when I try running the code blocks, I get an error. I tried fixing by adding a missing curly brace but then get another error. We may want to incorporate testing like @bknueven did for previous code blocks (although maybe we can get that through in another PR).